andela/ah-frontend-lannister
# #160617642 tagging articles

#### What does this PR do?
- This PR implements tagging functionality. This enables authors to create tags for the articles that they create.
- It also enables users to categorize articles by a certain tag.

#### Description of Task to be completed?

Tagging refers to the metadata that is assigned to a piece of content by the content creator and the readers/users of the content. Tagging allows for organization and or categorization of content.

#### How should this be manually tested?

- Enter a comma-separated string of words when creating an article and the words will be stored as tags
- Visit an article and you should be able to view tags for that article.
- Click the tags at the end of the article and you should be redirected to a page that shows articles under that tag.

#### Screenshots

#### Creating tags
<img width="1227" alt="screen shot 2018-11-21 at 18 12 24" src="https://user-images.githubusercontent.com/20440510/48850196-2af17500-edb9-11e8-8f35-51251381e9cf.png">

#### Viewing tags on an article

<img width="994" alt="screen shot 2018-11-21 at 18 12 45" src="https://user-images.githubusercontent.com/20440510/48850214-36dd3700-edb9-11e8-9728-2e57ca3f139e.png">

#### Viewing articles under a given tag

<img width="1102" alt="screen shot 2018-11-21 at 18 12 55" src="https://user-images.githubusercontent.com/20440510/48850248-49577080-edb9-11e8-9d5e-f33445e29e97.png">
